curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/access-multi-dimensional-arrays-with-indexes.english.md

```diff
@@ -23,7 +23,7 @@ arr[3][0]; // equals [10,11,12]
 arr[3][0][1]; // equals 11
 ```
 
-<strong>Note</strong><br>There shouldn't be any spaces between the array name and the square brackets, like <code>array [0][0]</code> and even this <code>array [0] [0]</code> is not allowed. Although JavaScript is able to process this correctly, this may confuse other programmers reading your code.
+<strong>Note</strong><br>There shouldn't be any spaces between the array name and the square brackets, like <code>array&nbsp;[0][0]</code> and even this <code>array&nbsp;[0]&nbsp;[0]</code> is not allowed. Although JavaScript is able to process this correctly, this may confuse other programmers reading your code.
```

Comment:
```suggestion
<strong>Note</strong><br>There shouldn't be any spaces between the array name and the square brackets, like `array [0][0]` and even this `array [0] [0]` is not allowed. Although JavaScript is able to process this correctly, this may confuse other programmers reading your code.
```
Nice catch!  Might as well use Markdown syntax, though - it's a little more concise.